lib/ansible/modules/network/enos/enos_config.py

```diff
@@ -184,9 +184,9 @@ def get_candidate(module):
     candidate = NetworkConfig(indent=1)
     if module.params['src']:
         try:
-            candidate.loadfp(module.params['src'])
-        except IOError:
             candidate.load(module.params['src'])
+        except IOError:
```

Comment:
It's required to handle exception here as it is already done in enos_config action plugin https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/action/enos_config.py#L102

Reply:
Resolving now